backend/app/services/implementations/sign_in_logs_service.py

```diff
@@ -24,9 +25,12 @@ def __init__(self, logger):
     def create_sign_in_log(self, user_id):
         sign_in = {"id": user_id, "time": datetime.now()}
         try:
-            new_sign_in = SignInLogs(**sign_in)
-            db.session.add(new_sign_in)
-            db.session.commit()
+            print("!!!!LADBLNALINPIWGAEN")
+            print(os.getenv("CREATE_SIGN_IN_LOG"))
+            if (os.getenv("CREATE_SIGN_IN_LOG")):
```

Comment:
So by default, environment variables are interpreted as strings, which are truthy. This means that this function will either return `"True"` or `"False"`, which will both be interpreted as `True`. So, you need to do something like this:

`os.getenv("CREATE_SIGN_IN_LOG") == "True"`